analyzers/src/SonarAnalyzer.CSharp/Rules/InterfaceMethodsShouldBeCallableByChildTypes.cs

```diff
@@ -75,10 +75,10 @@ protected override void Initialize(SonarAnalysisContext context)
             {
                 var identifierName = getIdentifierName(memberDeclaration);
 
-                analysisContext.ReportIssue(Diagnostic.Create(Rule,
-                                                                             identifierName.GetLocation(),
-                                                                             declaration.Identifier.ValueText,
-                                                                             string.Concat(explicitInterfaceSpecifier.Name, ".", identifierName.ValueText)));
+                analysisContext.ReportIssue(Rule,
+                                            identifierName,
+                                            declaration.Identifier.ValueText,
+                                            string.Concat(explicitInterfaceSpecifier.Name, ".", identifierName.ValueText));
```

Comment:
Whitespace: The `Rule` should be on the next line. Or everything should be on a single line if it fits into 200 chars